.github/ISSUE_TEMPLATE/bug_report_macos.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report (MacOS)
+description: File a bug report for MacOS.
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so that we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: os-version
+    attributes:
+      label: operating system version
+      description: |
+        Which version of MacOS do you use?
+        
+        You can follow [this link](https://support.apple.com/en-us/109033) to find out more.
+    validations:
+      required: true
+  - type: dropdown
+    id: proc-architecture
+    attributes:
+      label: Processor type
+      description: |
+        Which kind of processor do you use?
+
+        You can follow [this link](https://support.apple.com/en-us/109033) to find out more.
+      options:
+        - Intel
+        - Apple Silicon
```

Comment:
I've [seen](https://github.com/pymupdf/PyMuPDF/blob/4528c45b961b98dfcf97af98a71336dbd719fb58/.github/ISSUE_TEMPLATE/bug_report.yml?plain=1#L48-L63) other repos do this, in order to not have the "None" value show up:

```
-
- Intel
- Apple Silicon
```